scripts/issues.coffee

```diff
@@ -0,0 +1,43 @@
+# Description:
+#   Create GitHub issues for CH beheer and PC.COM. Please be concise.
+#
+# Dependencies:
+#   octonode
+#
+# Configuration:
+#   HUBOT_GITHUB_TOKEN = <accessToken>
+#
+# Commands:
+#   @pccom <query> - Creates a new issue and tags it 'pccom'
+#   @beheer <query> - Creates a new issue and tags it 'beheer'
+
+github = require 'octonode'
+
+githubToken = process.env.HUBOT_GITHUB_TOKEN;
+
+issue = null
+
+module.exports = (robot) ->
+  unless githubToken?
+    robot.logger.error "Missing HUBOT_GITHUB_TOKEN in environment"
+    return
+  robot.listen(
+    (message) ->
+      match = message.text?.match /^@(pccom|beheer) (?:me )?(.*)/i
+      if match
+        issue =
+          title: match[2],
+          body: "Issue added by #{message.user.name}"
+          labels: ['slack', match[1]]
+    (response) ->
+      # response.match is a boolean indicating whether the incoming message matched the defined pattern
+      # Ironically it doesn't exist when it it false. Great stuff.
```

Comment:
Dit fixt het:
```coffeescript
if match
  issue = ...
else
  false
```
https://github.com/github/hubot/blob/master/docs/patterns.md#dynamic-matching-of-messages